Makefile

```diff
@@ -31,9 +31,13 @@ test-commands:
 start-api:
 	docker-compose --project-name ${COMPOSE_PROJECT_NAME} up --build api
 
+# Starts the CLIENT application
+start-client:
+	docker-compose --project-name ${COMPOSE_PROJECT_NAME} up --build client
+
 # Start all the services.
 start:
-	docker-compose --project-name ${COMPOSE_PROJECT_NAME} up --build api
+	docker-compose --project-name ${COMPOSE_PROJECT_NAME} up --build
```

Comment:
Not totally sure if we want to force to build images on start every time. It may make sense or not.
Let me know your thoughts @tiagojsag

Reply:
depends on whether docker picks up changes automatically or not.

Reply:
It does in dev mode